subliminal/providers/addic7ed.py

```diff
@@ -126,7 +100,9 @@ def get_show_ids(self):
         soup = self.get('/shows.php')
         show_ids = {}
         for html_show in soup.select('td.version > h3 > a[href^="/show/"]'):
-            show_ids[html_show.string.lower()] = int(html_show['href'][6:])
+            show_ids[
+                IGNORED_CHARACTERS_RE.sub('', html_show.string).lower()] = \
+                    int(html_show['href'][6:])
```

Comment:
You should rename `IGNORED_CHARACTERS_RE` to something more cache-specific and create a `sanitize_cache` function (or similar) to put this logic.